src/_support/Page/Acceptance/Administrator/UserNotesFormPage.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * @package     Joomla.Test
+ * @subpackage  AcceptanceTester.Page
+ *
+ * @copyright   Copyright (C) 2005 - 2017 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+namespace Page\Acceptance\Administrator;
+
+/**
+ * Acceptance Page object class to user form page.
+ *
+ * @category  Users
+ * @package   Page\Acceptance\Administrator
+ * @copyright 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license   GNU General Public License version 2 or later; see LICENSE.txt
+ * @since     4.0.0
+ */
+class UserNotesFormPage
+{
+	/**
+	 * Subject
+	 *
+	 * @var    array
+	 * @since  4.0.0
+	 */
+	public static $subject = ['id' => 'jform_subject'];
+
+	/**
+	 * Select User Button
+	 *
+	 * @var    array
+	 * @since  4.0.0
+	 */
+	public static $selectUserButton = ['xpath' => '//*[@title="Select User"]'];
```

Comment:
avoid using * in xpaths, also could you please change this selector to something like this: 
`public static $selectUserButton = "//*[@title='Select User']";`
but please ensure you replace * used in xpath with the actuall html element, could be a div, p, button etc.